builds/build_python_runtime.sh

```diff
@@ -2,47 +2,47 @@
 
 set -euo pipefail
 
-# This is set by bob-builder from the "Build Path" comment line in the
-# build formula script that sourced this one.
-OUT_PREFIX=$1
-# The filename of the script that sourced this one (e.g. `python-3.10.0`).
-FORMULA_FILENAME=$(basename "${0}")
-# The version component (e.g. `3.10.0`).
-VERSION=$(echo "${FORMULA_FILENAME}" | cut --delimiter '-' --fields 2)
-
-echo "Building Python ${VERSION}..."
-
-if [[ "${STACK}" != "heroku-18" && "${STACK}" != "heroku-20" && "${VERSION}" == 3.[7-8].* ]]; then
-  echo "Python 3.7 and 3.8 are only supported on Heroku-20 and older!" >&2
-  echo "Override the default stacks list using: STACKS='heroku-18 heroku-20'" >&2
+PYTHON_VERSION="${1:?"Error: The Python version to build must be specified as the first argument."}"
+PYTHON_MAJOR_VERSION="${PYTHON_VERSION%.*}"
+
+INSTALL_DIR="/app/.heroku/python"
+SRC_DIR="/tmp/src"
+ARCHIVES_DIR="/tmp/upload/${STACK}/runtimes"
+
+echo "Building Python ${PYTHON_VERSION} for ${STACK}..."
+
+if [[ "${STACK}" != "heroku-18" && "${STACK}" != "heroku-20" && "${PYTHON_MAJOR_VERSION}" == 3.[7-8] ]]; then
+  echo "Error: Python ${PYTHON_MAJOR_VERSION} is only supported on Heroku-20 and older!" >&2
   exit 1
 fi
 
-# See: https://www.python.org/downloads/ -> "OpenPGP Public Keys"
-case "${VERSION}" in
-  3.1[0-1].*)
+# The release keys can be found on https://www.python.org/downloads/ -> "OpenPGP Public Keys".
+case "${PYTHON_MAJOR_VERSION}" in
```

Comment:
I assume that this naming is correct and `3.10` et.al are considered "major versions" even if the second part is usually denoting a minor version.